modules/wordads/wordads.php

```diff
@@ -466,11 +457,9 @@ function get_ad_snippet( $section_id, $height, $width, $location = '', $css = ''
 			'height'   => $height,
 		);
 		$ad_number   = count( $this->ads );
-		// Max 6 ads per page.
```

Comment:
Will it be okay from now on to have many ads on a given page? There should not be any more limits?

Reply:
I don't believe there will be: pafL3P-5e-p2 #comment-437

cc: @rclations for confirmation

Reply:
That's correct, the number of units is no longer limited in our back-end.